source/MaterialXTest/MaterialXRenderOgsFx/RenderOgsFx.cpp

```diff
@@ -181,7 +181,7 @@ bool OgsFxShaderRenderTester::runRenderer(const std::string& shaderName,
     return true;
 }
 
-TEST_CASE("Render: OgsFx TestSuite", "[renderglsl]")
+TEST_CASE("Render: OgsFx TestSuite", "[renderogsfx]")
```

Comment:
Separate out from general glsl tests. Will only run if the maya target/language is set.